stock_putaway_product_form/models/product_template.py

```diff
@@ -0,0 +1,27 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    product_tmpl_putaway_ids = fields.One2many(
+        comodel_name="stock.fixed.putaway.strat",
+        inverse_name="product_tmpl_id",
+        string="Product putaway strategies by product",
+    )
+
+    product_putaway_categ_ids = fields.Many2many(
+        comodel_name="stock.fixed.putaway.strat",
+        string="Product putaway strategies by category",
+        compute="_compute_putaway_categ_ids",
+    )
+
+    @api.depends("categ_id")
+    def _compute_putaway_categ_ids(self):
```

Comment:
The way to compute the putaway from categ is not correct.
Indeed if you look at the code here it will take the parent categ if not putway rule have been found for the categ
https://github.com/OCA/OCB/blob/12.0/addons/stock/models/product_strategy.py#L41
Be carefull it take only one categ per putaway strategie

Reply:
This has been changed